packages/web-components/src/components/card/card.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import { BASIC_COLOR_SCHEME } from '../../globals/shared-enums';
+import DDSLink from '../../globals/internal/link';
+import DDSCardFooter from './card-footer';
+import styles from './card.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The table mapping slot name with the private property name that indicates the existence of the slot content.
+ */
+const slotExistencePropertyNames = {
+  eyebrow: '_hasEyebrow',
+  heading: '_hasHeading',
+};
+
+/**
+ * Card.
+ * @element dds-card
+ */
+@customElement(`${ddsPrefix}-card`)
+class DDSCard extends DDSLink {
+  /**
+   * `true` if there is eyebrow content.
+   */
+  private _hasEyeblow;
+
+  /**
+   * `true` if there is heading content.
+   */
+  private _hasHeading;
+
+  /**
+   * `true` if there is copy content.
+   */
+  private _hasCopy;
+
+  /**
+   * Handles `slotchange` event.
+   */
+  private _handleSlotChange({ target }: Event) {
+    const { name } = target as HTMLSlotElement;
+    const hasContent = (target as HTMLSlotElement)
+      .assignedNodes()
+      .some(node => node.nodeType !== Node.TEXT_NODE || node!.textContent!.trim());
+    this[slotExistencePropertyNames[name] || '_hasCopy'] = hasContent;
+    this.requestUpdate();
+  }
+
+  /**
+   * @returns The inner content.
+   */
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
This method uses `this`, and thus below ESLint directive is not needed (my apologies if my prototype had that line).

```suggestion
```